content/docs/command-reference/metrics/index.md

```diff
@@ -103,8 +103,8 @@ to compare and pick the best performing experiment.
 ## Examples
 
 > This example is based on our
-> [Get Started](/doc/tutorials/get-started/experiments#project-metrics), where
-> you can find the actual source code.
+> [Get Started](/doc/start/experiments#collecting-metrics), where you can find
+> the actual source code.
```

Comment:
I see this addresses "Several cmd examples/links need updating" which is good although... Not related to 1.0 at all. Also, there's another 23 instances of old `tutorials/get-started` links that aren't  updated here...